src/web/mjs/connectors/To-Corona-Ex.mjs

```diff
@@ -21,29 +21,61 @@ export default class ToCoronaEx extends Connector {
     }
 
     async _getMangas() {
-        let uri = new URL('/comics?order=asc&sort=title_yomigana', this.apiurl);
+        let mangaList = [];
+        let nextCursor = '/comics?order=asc&sort=title_yomigana';
+        for(let run = true; run;) {
+            const data = await this._getMangasFromPage(nextCursor);
+            nextCursor = data.nextCursor;
+            mangaList.push(...data.chapters);
+            if(nextCursor == null) {
+                run = false;
+            }
+        }
+        return mangaList;
+    }
+
+    async _getMangasFromPage(nextCursor) {
+        let uri = new URL(nextCursor, this.apiurl);
         let request = new Request(uri, this.requestOptions);
         let data = await this.fetchJSON(request);
-        return data.resources.map(element => {
-            return {
-                id: element.id,
-                title: element.title.trim().replace('@COMIC', '')
-            };
-        });
+        return {
+            chapters: data.resources.map(element => {
```

Comment:
May be better to use `mangas:` instead of `chapters:`

Reply:
fixed